nxdrive/gui/application.py

```diff
@@ -225,6 +226,9 @@ def __init__(self, manager: "Manager", *args: Any) -> None:
         # to self.aboutToQuit can run beforehand.
         self.aboutToQuit.connect(self.manager.stop)
 
+        # Send previous crash metrics
+        self._send_crash_metrics()
```

Comment:
Out of curiosity, why did you move that logic from `Manager` to `Application`?

Reply:
If my memory is correct, it was because in `Manager.load()` the remotes inside the engines are not yes instanciated.